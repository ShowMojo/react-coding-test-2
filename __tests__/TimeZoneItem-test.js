import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import TimeZoneItem from '../app/components/timeZoneItem';

it('renders TimeZoneItem', () => {
  const tree = renderer
    .create(
      <TimeZoneItem
        timeZone={{
          id: 28,
          value: 'UTC-02',
          abbr: 'U',
          offset: -2,
          isdst: false,
          text: '(UTC-02:00) Coordinated Universal Time-02',
          zones: ['America/Noronha', 'Atlantic/South_Georgia', 'Etc/GMT+2'],
        }}
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
