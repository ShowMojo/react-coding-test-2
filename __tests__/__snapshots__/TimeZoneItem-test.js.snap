// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TimeZoneItem 1`] = `
<View
  style={
    Array [
      Object {
        "borderBottomColor": "#e9e9e1",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingHorizontal": 25,
        "paddingVertical": 30,
      },
    ]
  }
>
  <View
    style={
      Object {
        "width": "70%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
          },
          Object {
            "fontSize": 16,
            "fontWeight": "600",
            "marginBottom": 2,
          },
        ]
      }
    >
      UTC-02
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#828282",
          },
          Object {
            "fontSize": 12,
          },
        ]
      }
    >
      (UTC-02:00) Coordinated Universal Time-02
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "height": 32,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontSize": 30,
            "fontWeight": "bold",
          },
        ]
      }
    >
      03:22
    </Text>
    <Text
      style={
        Array [
          Object {
            "fontSize": 12,
            "fontWeight": "500",
            "marginTop": "auto",
          },
        ]
      }
    >
      PM
    </Text>
  </View>
</View>
`;
